docker-compose-index-m1.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.8"
+
+services:
+
+  elasticsearch:
+    build:
+      context: ./docker_compose/elasticsearch/
+    image: gds-elasticsearch-m1
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+    cap_add:
+      - IPC_LOCK
+    volumes:
+      - "./config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml"
+      - "./config/log4j2.properties:/usr/share/elasticsearch/config/log4j2.properties"
+      - "./config/jvm.options:/usr/share/elasticsearch/config/jvm.options"
+      - ./esdata:/usr/share/elasticsearch/data
+    ports:
+      - 9201:9200
```

Comment:
But why?

Reply:
That was to quickly take a peek at the ES while developing. Definitely not needed anymore. Will get rid of it.